### PR TITLE
Map RIGHT ALT/LANGUAGE SWITCH keys to SUPER key

### PR DESCRIPTION
**Pull request by tuanpham-dev**
In this PR, I added 2 options for mapping RIGHT ALT key to SUPER key and LANGUAGE SWITCH key to SUPER key.
- I think this is useful SUPER key can not be listened.
- The LANGAUGE SWITCH key is specifically for Samsung keyboard cover which is between SPACE and RIGHT ALT.

### CODE REVIEW

**Comment by gujjwal00** [comment]
Its late here so I will look more into this tomorrow, but two thoughts:

- Instead of checking every keycode, we can dynamically update [AndroidKeyCodeToXKeySym](https://github.com/gujjwal00/avnc/blob/24ec0e0b416b83a03c958072be42488b1d820ebc/app/src/main/java/com/gaurav/avnc/vnc/XKeySymAndroid.kt#L35) to set required key mapping. And IMO, we should do this update in Dispatcher, as other gestures are configured there and we may add more mappings in future.
- Can you please remove already merged commits from this PR? (Or I can cherry-pick the latest commit)

And thanks for this.

**Comment by tuanpham-dev** [comment]
@gujjwal00 Thanks for your feedbacks. I have updated the code as well as the commit. Please check it again and let me know your thoughts.

**Comment by gujjwal00** [comment]
I might not have explained properly what I was thinking, and you were right to change `KeyHandler` (sorry! I was half-asleep at that point).

[This approach](https://github.com/gujjwal00/avnc/commit/51494c5e93904f3249779b23875a05ae59e627fc) is what I was thinking about.

This way we update the key-mappings at `KeyHandler` construction, and they are automatically used by rest of the code. Now we don't have to have to check every event. It is also simpler to add new mappings in future.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
     <string name="pref_viewer">Viewer</string>
     <string name="pref_viewer_summary">Picture-in-Picture, Zoom</string>
     <string name="pref_input">Input</string>
-    <string name="pref_input_summary">Gestures, Mouse, Virtual keys</string>
+    <string name="pref_input_summary">Gestures, Mouse, Virtual keys, Key Mappings</string>
```
Comment (gujjwal00) [inline app/src/main/res/values/strings.xml:99]: Mentioning 'Key Mapping' is not necessary here. It causes the summery to wrap-around to second line on some smaller devices. It also makes too long with respect to other summaries.

**Comment by tuanpham-dev** [comment]
Got it, feel free to edit my commit or I will do it when I have some spare time.

**Comment by gujjwal00** [comment]
Merged with changes.
Updated the preference category icon to full keyboard, to differentiate it from Virtual Key icon.

Again, I am sorry for wasting your time earlier, should not have rushed the review.

**Pull request merged**